.idea/modules.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should not be commited. Please add it to the .gitignore. I think we should simply add .idea and .idea/** to .gitignore, so that no config files from this directory are commited.

Reply:
done